src/tools/kata-ctl/src/args.rs

```diff
@@ -100,7 +100,41 @@ pub struct IptablesCommand {
 #[derive(Debug, Subcommand)]
 pub enum IpTablesArguments {
     /// Configure iptables
-    Metrics,
+    #[derive(Debug, Args)]
+pub struct IptablesCommand {
```

Comment:
It looks like this might be a copy/paste error as this type is already defined on line 95 above.

Reply:
Hi @pembek01 and @tg5788re - the code still doesn't build and this is one of the problems. It looks like you need to delete lines 102-105 inclusive as the `IpTablesArguments` type is already defined above (lines 94-98 inclusive).